rlp/decode.go

```diff
@@ -48,6 +49,14 @@ var (
 	errUintOverflow  = errors.New("rlp: uint overflow")
 	errNoPointer     = errors.New("rlp: interface given to Decode must be a pointer")
 	errDecodeIntoNil = errors.New("rlp: pointer given to Decode must not be nil")
+
+	//Stream pool
```

Comment:
I don't think you need this comment here, it's not providing any more info than the variable name.

Reply:
fixed